src/rebar3_hex.erl

```diff
@@ -10,7 +10,9 @@
         , help_opt/0
         ]).
 
--type task() :: #{args := map(), 
+-type task() :: #{raw_opts := proplists:proplist(),
```

Comment:
These names don't make a lot of sense, but it's internal so it should be fine for now.